pkg/lib/indexer/indexer.go

```diff
@@ -1,45 +1,54 @@
 package indexer
 
 import (
+	"context"
+	"database/sql"
+	"fmt"
 	"io"
 	"io/ioutil"
 	"os"
 	"path"
+	"path/filepath"
 
 	"github.com/operator-framework/operator-registry/pkg/containertools"
+	"github.com/operator-framework/operator-registry/pkg/lib/bundle"
 	"github.com/operator-framework/operator-registry/pkg/lib/registry"
+	pregistry "github.com/operator-framework/operator-registry/pkg/registry"
```

Comment:
it's not for this PR to fix, but this module layout is confusing

Reply:
agreed